packages/core/src/electron-main/electron-main-application.ts

```diff
@@ -182,6 +183,10 @@ export class ElectronMainApplication {
     readonly backendPort = this._backendPort.promise;
 
     protected _config: FrontendApplicationConfig | undefined;
+    protected frame: boolean = true;
```

Comment:
Another case where I'll dare to disagree with the established terminology. I think this would be clearer as something like `useNativeWindowFrame`